components/expression/expression-renderer.js

```diff
@@ -113,19 +113,19 @@ ExpressionRenderer.prototype.initLayout = function() {
 };
 
 /** @inheritDoc */
-ExpressionRenderer.prototype.layout = function() {
+ExpressionRenderer.prototype.layout = function () {
   // Gets the label sizes so as to set the offset of heatmap SVG.
   this.getHeatmapLabelSizes_();
   // Compute the shifting sizes.
   this.profileHeight_ = this.data.options.showProfiles ?
-      this.DEFAULT_PROFILE_HEIGHT : 0;
```

Comment:
If the line continues from the previous line, and breaks at an operator, we use 4 indent instead of 2.